### DISCUSSION TITLE
wallpaper no longer works, the images remain fixed..

### DISCUSSION THREAD

**Discussion by kris06** [jurialmunkey/skin.arctic.horizon.2#962] [Discussion]
arctic horizon 2 . For a few days, the dynamic images in the background no longer work, this remains fixed..

**Comment by jurialmunkey** [comment]
Open an issue to report bugs.
https://github.com/jurialmunkey/skin.arctic.horizon.2/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml&title=%5BBUG%5D+Short+Description
